### ISSUE TITLE
Trying to access array offset on value of type null

### ISSUE THREAD

**Issue by alexgarrettsmith** [torchlight-api/torchlight-commonmark-php#3]
I haven't had much time to dig into this issue, but filing this now before I do 👍

When attempting to render content with `Markdown::convertToHtml($this->body)` (using the https://github.com/GrahamCampbell/Laravel-Markdown package).

```
Trying to access array offset on value of type null
```

Stacktrace shows the error happening here:

```php
protected function getLanguage($node)
{
    $language = $this->getInfo($node)[0]; // here

    return $language ? Xml::escape($language, true) : null;
}
```

With the following content in its entirety (this is user generated content that *should* have backticks around the code blocks).

```
ok I'm stumped on this error - anyone have any ideas? I'm on lesson 20. files.js:

import axios from 'axios'

export default {
    namespaced: true,
    state: {
        files: []
    },
    mutations: {
        SET_FILES(state, files) {
            state.files = files
        },

        ADD_FILE(state, file) {
            state.files = [file, ...state.files]
        }        
    },
    getters: {
        files(state) {
            return state.files
        }
    },
    actions: {
        async getFiles({ commit }) {
            let response = await axios.get('/api/files')

            commit('SET_FILES', response.data)
        }
    },
  }

Upload.vue

import { mapActions, mapGetters, mapMutations } from "vuex"
import AppFiles from '../components/AppFiles.vue'
import AppUploader from '../components/AppUploader.vue'
import axios from 'axios'
export default {
	components: {
		AppFiles,
		AppUploader
	},	
	computed: {
		...mapGetters({
			files: 'files/files'
		})
	},

	methods: {
		...mapActions({
			getFiles: 'files/getFiles'
		}),

		...mapMutations({
			addFile: 'files/ADD_FILE'
		}),
		async storeFile(file) {
			let response = await axios.post('/api/files', {
				name: file.filename,
				size: file.fileSize,
				path: file.serverId
			})		

			this.addFile(response.data.data)
		}
	},
	mounted() {
		this.getFiles()
	}
};

error:

nonIterableSpread.js?3427:2 Uncaught (in promise) TypeError: Invalid attempt to spread non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.
    at _nonIterableSpread (nonIterableSpread.js?3427:2)
    at _toConsumableArray (toConsumableArray.js?2909:6)
    at Store.ADD_FILE (files.js?d979:14)
    at wrappedMutationHandler (vuex.esm-browser.js?5502:808)
    at commitIterator (vuex.esm-browser.js?5502:438)
    at Array.forEach (<anonymous>)
    at eval (vuex.esm-browser.js?5502:437)
    at Store._withCommit (vuex.esm-browser.js?5502:596)
    at Store.commit (vuex.esm-browser.js?5502:436)
    at Store.boundCommit [as commit] (vuex.esm-browser.js?5502:376)
```

I'm currently handling this using the `rescue` helper in Laravel, to default to the unmarkdown'd value.

```php
'body_markdown' => rescue(fn () => Markdown::convertToHtml($this->body), $this->body),
```

I'll do some digging with this specific case and PR if needed.

**Comment by aarondfrancis** [comment]
@alexgarrettsmith Thank you for the detailed report! I'll look into this tonight and see if I can ship something.

**Comment by aarondfrancis** [comment]
I just published `0.5.2` which I *think* should fix it. I was able to get it to fail reliably using indented code, so I added a test and fix for it. 

Let me know if that does it!

I think you're the first to use it for UGC, so I'll be curious what else you run into.

**Comment by alexgarrettsmith** [comment]
`0.5.2` has done the trick, thanks Aaron 👍

I'll keep the `rescue` there just in-case, but I'll get exceptions logged. I'll add an issue if anything else comes up with UGC!
